lib/Stripe/BitcoinTransaction.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class Stripe_BitcoinTransaction extends Stripe_ApiResource
```

Comment:
I chatted about it with @raycmorgan and @canderson and we felt it was an unnecessarily granular unit to expose at the bindings level. It represents the set of transactions that fill a bitcoin receiver (customers pushing coins to the receiver) and we couldn't think of much of a valid use case for querying those outside the context of a bitcoin receiver. The proposed changes here does allow them to be pulled as a sublist

```php
$receiver->transactions->all();
```

How do you feel about this approach?